### ISSUE TITLE
testing broken

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#1378]
currently our container brings python 3.10.2 and gets downgraded to 3.9.10. 

This creates problems.

```
Preparing transaction: ...working... done
1011
Verifying transaction: ...working... done
1012
Executing transaction: ...working... done
1013
/opt/conda/envs/mssenv/lib/python3.9/site-packages/_distutils_hack/__init__.py:30: UserWarning: Setuptools is replacing distutils.
1014
  warnings.warn("Setuptools is replacing distutils.")
1015
Traceback (most recent call last):
1016
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/conda/gateways/connection/__init__.py", line 21, in <module>
1017
    from requests import ConnectionError, HTTPError, Session
1018
ModuleNotFoundError: No module named 'requests'
1019

1020
During handling of the above exception, another exception occurred:
1021

1022
Traceback (most recent call last):
1023
  File "/opt/conda/envs/mssenv/bin/mamba", line 7, in <module>
1024
    from mamba.mamba import main
1025
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/mamba/mamba.py", line 22, in <module>
1026
    from conda.cli.install import check_prefix, clone, get_revision
1027
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/conda/cli/install.py", line 19, in <module>
1028
    from ..core.index import calculate_channel_urls, get_index
1029
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/conda/core/index.py", line 13, in <module>
1030
    from .package_cache_data import PackageCacheData
1031
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/conda/core/package_cache_data.py", line 14, in <module>
1032
    from .path_actions import CacheUrlAction, ExtractPackageAction
1033
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/conda/core/path_actions.py", line 30, in <module>
1034
    from ..gateways.connection.download import download
1035
  File "/opt/conda/envs/mssenv/lib/python3.9/site-packages/conda/gateways/connection/__init__.py", line 39, in <module>
1036
    from pip._vendor.requests import ConnectionError, HTTPError, Session
1037
ModuleNotFoundError: No module named 'pip'
1038
Error: Process completed with exit code 1.
```

**Comment by ReimarBauer** [comment]
Discussed on conda-forge with Wolf Vollbrecht, will be fixed upstream

**Comment by ReimarBauer** [comment]
https://github.com/mamba-org/mamba/pull/1545

solved by libmamba 0.22.1

https://github.com/mamba-org/mamba/commit/4b663ada019715ad4c989a1a57e81fe12edde3d8


Thx
